src/components/Search.js

```diff
@@ -278,8 +288,9 @@ class Search extends Component {
                                                 preferences[listComponent],
                                             )}
                                             dataField={`${
-                                                preferences[listComponent]
-                                                    .dataField
+                                                preferences[
+                                                    listComponent
+                                                ].dataField.split('.')[0]
```

Comment:
@metagrover This should fix the issue, of saved preferences already having `keyword` in `dataField`. Thoughts ?Not tested as I have new default Preference, and this works well with it.